package.json

```diff
@@ -24,10 +27,11 @@
     "eslint-config-openlayers": "4.0.0",
     "jquery": "2.2.0",
     "less": "git://github.com/less/less.js#ec04a03f1cba3a092d5cd7f7c5d8e28bb43c1932",
+    "openlayers": "git://github.com/openlayers/ol3#9baa296a49a75045e756089516f610c58cdc82eb",
```

Comment:
Do you really need to change OL version?
I have already updated it in the "advanced search" PR.